tests/kani/FunctionCall_MultiModuleSameHarnessName/main.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
```

Comment:
Can you please add a comment on what this is testing?

Reply:
@celinval Resolved in https://github.com/model-checking/kani/pull/1955/commits/87d1214108935310626f5b6bea1e5cb169c32acf